.github/workflows/airac.yml

```diff
@@ -1,40 +1,52 @@
-name: Automatic AIRAC Actions
+name: Automatic AIRAC actions
 
 on:
   repository_dispatch:
     types: [auto_airac_actions]
   workflow_dispatch:
 
 jobs:
-  python-script-and-pr:
+  airac:
+    name: Perform AIRAC actions
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Set up Git
+        run: |
+          git config --local user.email "privileged-access@vatsim.uk"
+          git config --local user.name "VATSIM UK"
+
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: '3.x'
+          python-version: "3.x"
 
       - name: Install dependencies
-        run: pip install -r .github/workflows/requirements.txt
+        run: pip install -r workflows/airac/requirements.txt
 
-      - name: Run Python script
+      - name: Update sector file and GNG data
         env:
-            REMOTE_KEY: ${{ secrets.GITHUB_TOKEN }}
+          REMOTE_KEY: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          python .github/workflows/auto_airac_actions.py
+          python workflows/airac/auto_airac_actions.py
           echo "airac=$(cat airac.txt)" >> "$GITHUB_ENV"
           rm airac.txt
 
       - name: Commit changes
         run: |
-          git config --local user.email "privileged-access@vatsim.uk"
-          git config --local user.name "VATSIM UK"
           git add .
-          git commit -m "Sector file update for AIRAC ${{ env.airac }}"
```

Comment:
Reason for deleting these lines?

Reply:
The `git config` lines have been moved further up the file, as they are common with both commits. The `git commit` line is on the line below, with an updated message.